crates/cli/src/subcommands/publish.rs

```diff
@@ -24,12 +25,12 @@ pub fn cli() -> clap::Command {
                 .help("The type of host that should be for hosting this module"),
         )
         .arg(
-            // TODO(jdetter): Rename this to --delete-tables (clear doesn't really implies the tables are being dropped)
-            Arg::new("clear_database")
-                .long("clear-database")
+            Arg::new("destroy_previous")
```

Comment:
let's keep this as `clear-database` - I realized that this was us trying to not break backwards compatibility but let's just break the API in this case.

Reply:
I actually renamed this because of the `TODO` above - I'm fine to revert, but I do agree with it in principle!